### ISSUE TITLE
Can't start an Angular project if Angular CLI can't be found, despite setting angularProcessCommand

### ISSUE THREAD

**Issue by nicole-ashley** [lucono/karma-test-explorer#34]
**Describe the Bug**

Because the check for the Angular CLI module path https://github.com/lucono/karma-test-explorer/blob/ec4a6c553b024f325d50db042cf7cad63ad2ffba/src/frameworks/angular/angular-test-server-executor.ts#L59-L64 happens before the check for `options.angularProcessCommand` https://github.com/lucono/karma-test-explorer/blob/ec4a6c553b024f325d50db042cf7cad63ad2ffba/src/frameworks/angular/angular-test-server-executor.ts#L80-L85 if the CLI path can't be found there's no way to provide a manual command to work around this.

This is an issue with projects that use Yarn PNP or any other kind of PNP package manager where `get-existing-path` can't find the module because it only looks for a `node_modules` folder, which doesn't exist for these package managers. 

Alternatively (better, but more work), you could probably replace your usages of `get-existing-path` with `require.resolve` which works fine with these package managers.

**Steps to Reproduce the Behavior**

1. Start a new project with Yarn 3 or later
2. Set `karmaTestExplorer.angularProcessCommand` to `yarn ng`
3. Try to use this extension

**Describe the Expected Behavior**

- The extension works normally

**Provide Debug Logs**

```[2022-07-08 02:15:31.814] [INFO] [KarmaTestListener]: Waiting for Karma to connect on port 9999
[2022-07-08 02:15:31.889] [ERROR] [DefaultTestManager]: System component terminated with message: Karma server failed to start: Angular CLI does not seem to be installed - You may need to run 'npm install' in your project. Please install it and try again.
[2022-07-08 02:15:31.908] [ERROR] [DefaultTestManager]: Error: Karma server failed to start: Angular CLI does not seem to be installed - You may need to run 'npm install' in your project. Please install it and try again.
[2022-07-08 02:15:31.909] [ERROR] [Adapter]: Failed to load tests - Karma server failed to start: Angular CLI does not seem to be installed - You may need to run 'npm install' in your project. Please install it and try again.
[2022-07-08 02:15:31.913] [INFO] [Adapter]: Finished loading tests in 0.22 secs (Failed)```

**Please Provide the Following Information**
 
 - OS: any
 - Environment: local/wsl
 - Node Version: lts w/ yarn3
 - VSCode Version: any
 - Karma Version:any
 - Frameworks: Angular

**Comment by lucono** [comment]
This is a bug, thanks for reporting it. I'll plan to release a fix.

**Comment by nicole-ashley** [comment]
Thanks @lucono!

**Comment by lucono** [comment]
This should be fixed in [v0.7.0](https://github.com/lucono/karma-test-explorer/blob/master/CHANGELOG.md#070---jul-17-2022). Please update to the new version and confirm.

**Comment by lucono** [comment]
Closing this - please open a new issue if the problem still exists.
